### DISCUSSION TITLE
Immich go utility failing for google takeout files

### DISCUSSION THREAD

**Discussion by grpursuing** [immich-app/immich#7868] [General]
Trying to upload google take out with Immich go but most of the files are failing with error - JSON file not found for this file. I had 3 takeout files in total and 1 of them successfully got loaded but having this issue with most of the 2nd file. - Any pointers ??

**Comment by alextran1502** [comment]
You should open the issue over at immich-go repo to get help

**Reply by simulot** [reply]
Have you processed all parts of the archive together? Images and Json can be placed in different part of the archive set.

**Comment by grpursuing** [comment]
Ah i did not, how do i put everything together?Regards,GouravSent from my iPhoneOn Mar 12, 2024, at 4:19 PM, Jean François CASSAN ***@***.***> wrote:﻿
Have you processed all parts of the archive together? Images and Json can be placed in different part of the archive set.

**Reply by simulot** [reply]
You have to use a wild char as file name like `-upload -google-photos /path/to/files/takeout*.zip`

**Reply by simulot** [reply]
You can read some explanation on the immich-go readme:
https://github.com/simulot/immich-go#google-photos-best-practices
